tests/bin/bootstrap.test.ts

```diff
@@ -2,15 +2,17 @@ import os from 'os';
 import path from 'path';
 import fs from 'fs';
 
-import { checkKeynodeState } from '@/bootstrap';
+import { isDirectoryClear } from '@/bootstrap';
 
 import * as utils from './utils';
 
-describe.skip('CLI bootstrap', () => {
+describe('CLI bootstrap', () => {
   let dataDir: string;
   let passwordFile: string;
   let nodePath: string;
 
+  process.env.PK_FAST_KEY_GEN = 'true';
```

Comment:
Yea these sort of variables I would want to remove. I can think we can replace this in our tests with directly providing a pre-generated recovery key.